bagelbids/tests/data/example2.json

```diff
@@ -1,43 +1,66 @@
 {
-    "participant_id": {
-        "Description": "A participant ID",
-        "Annotations": {
-            "IsAbout": {
-                "TermURL": "nb:ParticipantID",
-                "Label": "Unique participant identifier"
-            }
-        }
+  "participant_id": {
+    "Description": "A participant ID",
+    "Annotations": {
+      "IsAbout": {
+        "TermURL": "bg:ParticipantID",
+        "Label": "Unique participant identifier"
+      }
+    }
+  },
+  "session_id": {
+    "Description": "A session ID",
+    "Annotations": {
+      "IsAbout": {
+        "TermURL": "bg:SessionID",
+        "Label": "Unique session identifier"
+      }
+    }
+  },
+  "group": {
+    "Description": "Group variable",
+    "Levels": {
```

Comment:
The short answer is: so we can still play with BIDS. The BIDS `Levels` can only have one string value per level ([see here](https://bids-specification.readthedocs.io/en/stable/glossary.html#levels-metadata)) but our `Levels` need to have object values that contain (for now) `TermURL` and `Label`. It is possible (even likely) that the "BIDS" human-readable label and our human-readable label will be different. 

We had a lengthy conversation with the Repronim folks about where in the BIDS data dictionary we can stick our extended semantic annotations (or if they should live in a separate file alltogether). Every choice will be some compromise, but I think what we are doing now is the most usable, both for a BIDS user / developer and us.

That said, I think I should update the [docs here](https://www.neurobagel.org/documentation/dictionaries/#sex) to show that we will indeed retain the BIDS Levels and our own Levels because I am sure this question will come up again!